File: js/palindrome.js
exports.palindrome = function(word) {
    if(isNaN(word)){
        let newWord = word.toLowerCase()
        let testWord = newWord.split('').reverse('').join('')
        return (newWord == testWord)? true: false     
    }else{
        let numWord = word.toString()
        let testNumWord = numWord.split('').reverse('').join('')
        return (numWord == testNumWord)? true : false
    } 
}

File: python/palindrome.py
from audioop import reverse
from cmath import isnan, nan


def palindrome(word):
    word = str(word)
    lower_case_word = word.lower()
    # print(len(lower_case_word))
    # reverse_word = lower_case_word[::-1]
    reversed_word = ''.join(reversed(lower_case_word))
    # print((reverse_word))

    if lower_case_word == reversed_word:
        return True
    else:
        return False

print(palindrome('racecar') == True)
print(palindrome('Noon') == True)
print(palindrome('ciVic') == True)
print(palindrome('nice') == False)
print(palindrome(434) == True)
print(palindrome(123) == False)
print(palindrome('bomb') == False)